operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -87,14 +87,11 @@
         "y": 0
       },
       "id": 93,
+      "repeat": "datasource",
       "title": "Alerts",
       "type": "row"
     },
     {
-      "datasource": {
-        "type": "prometheus",
-        "uid": "${DS_PROMETHEUS-APPSTUDIO-DS}"
```

Comment:
@Roming22  - we should in fact revert that change in your validation script, and no longer allow 

`"uid": "${DS_PROMETHEUS-APPSTUDIO-DS}"` 

as this at least with my dev_setup.sh testing was what caused the alert panels to not rendor automatically

Reply:
in fact @Roming22 we should be preventing any `"datasource":{...}` entries

Reply:
in favor of the ` "repeat": "datasource",` entry in the row the panel is a part of

Reply:
Sure. Let's connect to make sure I understand the requirements and make that happen.